cineast-core/src/main/java/org/vitrivr/cineast/core/features/OCRSearch.java

```diff
@@ -25,3 +79,221 @@ protected String enrichQueryTerm(String queryTerm) {
     return queryTerm + "~1";
   }
 }
+
+class TesseractRecognizer {
```

Comment:
Please make sure there's only one class per file, unless there is a very good reason. You can extract these classes to a separate file.

Reply:
I moved the other classes inside the main SingleFrameOCRSearch class with my last commit. Will that be alright?

Reply:
Please only use one class per file, unless there is a very good reason. I'm not seeing a good reason to have multiple classes inside the SingleFrameOCRSearch.

Reply:
Alright, I'll get it sorted out as soon as possible.